### PR TITLE
Ensure the correct pipeline file is present

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Moves the correct pipeline file over from bugsnag-js

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -1,171 +1,4 @@
...
-
-  - label: ':docker: Build and publish JS packages'
-    key: 'publish-js'
-    timeout_in_minutes: 30
-    plugins:
-      - docker-compose#v3.9.0:
-          build: publisher
-          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
-    env:
-      BUILD_RN_WITH_LATEST_NATIVES: ${BUILD_RN_WITH_LATEST_NATIVES}
```
Comment (imjoehaines) [inline .buildkite/pipeline.yml:47]: The "trigger Expo pipeline" step has `depends_on: 'publish-js'`, so I assume this is still needed ?

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,171 +1,4 @@
...
-  #
-  # License audit
-  #
-  - label: ':copyright: License Audit'
-    timeout_in_minutes: 20
-    agents:
-      queue: opensource-mac-cocoa-11
-    env:
-      DEVELOPER_DIR: "/Applications/Xcode12.app"
-    command: scripts/license_finder.sh
```
Comment (imjoehaines) [inline .buildkite/pipeline.yml:11]: AFAIK we still need this in every repo

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -1,171 +1,4 @@
...
-    command: 'npm run test:lint'
-
-  - label: 'Unit tests'
-    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'ci-image'
-    timeout_in_minutes: 10
-    plugins:
-      docker-compose#v3.9.0:
-        run: ci
-    command: 'npm run test:unit'
```
Comment (imjoehaines) [inline .buildkite/pipeline.yml:158]: This is all still needed as well

**Review by imjoehaines — CHANGES_REQUESTED** [review]

**Comment by Cawllec** [comment]
This change is unnecessary, and can be closed

**Pull request closed without merging**